packages/bytes/test/byte_stream_chunked_reader_spec.savi

```diff
@@ -294,24 +279,24 @@
     stream.mark_here
     stream.advance_to_end
 
-    assert: try (stream.read_native_u16!(0) == 0x2301 | False)
-    assert: try (stream.read_native_u16!(1) == 0x4523 | False)
-    assert: try (stream.read_native_u32!(0) == 0x67452301 | False)
-    assert: try (stream.read_native_u32!(1) == 0x89674523 | False)
-    assert: try (stream.read_native_u64!(0) == 0xEFCDAB8967452301 | False)
-    assert: try (stream.read_native_u64!(1) == 0x01EFCDAB89674523 | False)
-
-    assert: try (stream.read_le_u16!(0) == 0x2301 | False)
-    assert: try (stream.read_le_u16!(1) == 0x4523 | False)
-    assert: try (stream.read_le_u32!(0) == 0x67452301 | False)
-    assert: try (stream.read_le_u32!(1) == 0x89674523 | False)
-    assert: try (stream.read_le_u64!(0) == 0xEFCDAB8967452301 | False)
-    assert: try (stream.read_le_u64!(1) == 0x01EFCDAB89674523 | False)
-
-    assert: try (stream.read_be_u16!(0) == 0x0123 | False)
-    assert: try (stream.read_be_u16!(1) == 0x2345 | False)
-    assert: try (stream.read_be_u32!(0) == 0x01234567 | False)
-    assert: try (stream.read_be_u32!(1) == 0x23456789 | False)
-    assert: try (stream.read_be_u64!(0) == 0x0123456789ABCDEF | False)
-    assert: try (stream.read_be_u64!(1) == 0x23456789ABCDEF01 | False)
+    assert try: stream.read_native_u16!(0) == 0x2301
+    assert try: stream.read_native_u16!(1) == 0x4523
+    assert try: stream.read_native_u32!(0) == 0x67452301
+    assert try: stream.read_native_u32!(1) == 0x89674523
+    assert try: stream.read_native_u64!(0) == 0xEFCDAB8967452301
+    assert try: stream.read_native_u64!(1) == 0x01EFCDAB89674523
```

Comment:
Hmmmm... The tests that were set up like this are now not actually testing what they were meant to test anymore.

Previously we had them set up like this:
```
assert: try (stream.read_native_u16!(0) == 0x2301 | False)
```

So that if there was an error, the test would fail due to the explicit `False`.

And if the `==` comparison failed, the test would fail due to `==` returning false.

But, now we have this:
```
assert try: stream.read_native_u16!(0) == 0x2301
```

Which will fail the test if an error is raised, but I think it _won't_ fail the test anymore if the `==` comparison returns false.

---

I'm not sure what the ergonomic way of solving this is. We'll have to brainstorm it a bit.